### ISSUE TITLE
[Q] - 

### ISSUE THREAD

**Issue by doctorleach** [jitesoft/docker-tesseract-ocr#3]
I downloaded the image and used the recommended commands (I put in my own image info of course):

```
docker pull jitesoft/tesseract-ocr
docker run -v /path/to/image/img.jpg:/tmp/img.jpg jitesoft/tesseract-ocr /tmp/img.jpg stdout
```

However, this results in the output:

```
Error opening data file /usr/local/share/tessdata/eng.traineddata
Please make sure the TESSDATA_PREFIX environment variable is set to your "tessdata" directory.
Failed loading language 'eng'
Tesseract couldn't load any languages!
Could not initialize tesseract.
```
If I attach the container and look inside, there is no /usr/local/share/tessdata directory. But if I run the container with  --list-langs I get

```
List of available languages in "/usr/local/share/tessdata/" (3):
eng
equ
osd
```

Is there a step missing from the instructions?  I'm running it on a Mac Studio

Tack tack

**Comment by Johannestegner** [comment]
Hi there!

Thank you for the report!

It seems that the language training files have changed URI, so the image fails to download the correct training data and instead downloads a 404 page or something.

Fixing it right now and will publish a new image as soon as it finishes building!

**Comment by Johannestegner** [comment]
Fix applied in 18d062198a8a2a4e30d355d3e7787ea24370bc4d
Building as we speak.

**Comment by Johannestegner** [comment]
Build completed and tested, working as intended now.

Sorry for the inconvenience and again, thank you for the report :)
